components/skeleton/Button.tsx

```diff
@@ -4,14 +4,14 @@ import classNames from 'classnames';
 import SkeletonElement, { SkeletonElementProps } from './SkeletonElement';
 import { ConfigConsumer, ConfigConsumerProps } from '../config-provider';
 
-interface SkeletonButtonProps extends Omit<SkeletonElementProps, 'size'> {
+interface ButtonProps extends Omit<SkeletonElementProps, 'size'> {
```

Comment:
Still keep `SkeletonButtonProps` since `ButtonProps` should used for Button component.

Reply:
reverted.